version.h

```diff
@@ -0,0 +1,14 @@
+/**
+ * @file version.h
```

Comment:
Can we rename it `lv_version.h`?

Reply:
Should be addressed in https://github.com/lvgl/lvgl/compare/503a63ac7687019d02ef0c0a98ccc2ea9e0fb190..630ea60294159d4100101eee9e18e2dbd77597a1 , the previous push is just a rebase on 9.1.1 .